Directory.Build.props

```diff
@@ -0,0 +1,31 @@
+<Project>
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AvaloniaVersion>11.0.1</AvaloniaVersion>
```

Comment:
May be we should keep packages settings in Directory.Packages.props?

Reply:
No problem